nix/modules/blocksense/backends/process-compose.nix

```diff
@@ -1,17 +1,21 @@
-{ self', cfg, ... }:
+{
+  self',
+  cfg,
+  ...
+}:
 {
   config,
   lib,
   ...
 }:
 let
   inherit (self'.apps)
-    sequencer
-    blocksense
     blockchain_reader
     aggregate_consensus_reader
     ;
 
+  mkCargoTargetExePath = executable-name: "${config.devenv.root}/target/release/${executable-name}";
```

Comment:
How do I produce the release build?
cargo build --release ?

For example, I have run just clean and lost my target folder. And I expect direnv reload & process-compose to do the work. But I see this log from the sequencer

```
bash: line 1: /home/emilivanichkov/code/repos/metacraft-labs/blocksense/monorepo/target/release/sequencer: No such file or directory
```

In the PR description, there are no instruction for the new workflow, nor there are such in our discord. Please provide